django_netjsongraph/base/node.py

```diff
@@ -114,9 +81,11 @@ def get_from_address(cls, address, topology):
         :param topology: Topology instance
         :returns: Node object or None
         """
-        address = ';{0};'.format(address)
-        return cls.objects.filter(topology=topology,
-                                  addresses__contains=address).first()
+        objs = list(cls.objects.filter(addresses__contains=address))
```

Comment:
what happens if the address we're testing is `192.168.1.1` but there's an address which has `192.168.1.1`?

Reply:
I have added check for this below this this line.

Reply:
```
        for obj in objs:
            if address in obj.addresses:
                return obj
        return None
```
This below these lines make sure that any such entries are removed.